docker/containerbuild.sh

```diff
@@ -32,7 +32,17 @@ if [ -d "$BUILDROOT" ]; then
 fi
 
 # Fetch the boot.iso for the build.
-curl http://mirror.centos.org/centos/"${KSNAME##*-}"/os/x86_64/images/boot.iso -o /tmp/boot-"${KSNAME##*-}".iso
+if [[ ${CONT_ARCH} -eq "ppc64le" ]]; then
+    VERSION=`expr "${KSNAME}" : '.*\-\([0-9\.]*\).*'`
+    if [[ "${VERSION}" = "7" ]]; then
+        curl http://mirror.centos.org/altarch/"${VERSION}"/isos/ppc64le/CentOS-7-AltArch-ppc64le-NetInstall-1511.iso -o /tmp/boot-"${KSNAME##*-}".iso
```

Comment:
we should not lock version and filename down, this will break for any change or update in media. can we make it generic ?